app/views/application_records/show.haml

```diff
@@ -1,15 +1,16 @@
-%h1 Application Record
+%h1.h2="#{@record.position.name} Application Record"
```

Comment:
Except that's not what those classes are for in Bootstrap - [they're about maintaining the font size while having a different display style](http://getbootstrap.com/css/#type-headings), not having an `h1` that is the same size as an `h2`.